src/ModelContextProtocol-SemanticKernel/Types/Vscode/VsCodeSettings.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Text.Json.Serialization;
+
+namespace ModelContextProtocol.SemanticKernel.Types;
+
+/// <summary>
+/// Wrapper for deserialization
+/// </summary>
+internal class VsCodeSettings
```

Comment:
Sometimes you use `VS` and sometimes `Vs`, can you please use 1 naming.

Can you fix this in: 
- the class names 
- the file names
- the folder names